plugins/verifier/sbom/sbom.go

```diff
@@ -126,3 +182,39 @@ func processSpdxJSONMediaType(name string, refBlob []byte) (*verifier.VerifierRe
 		Message:   fmt.Sprintf("SBOM failed to parse: %v", err),
 	}, err
 }
+
+// iterate through all package info and check against the deny list
+// return the violation packages
+func filterDisallowedPackages(packageLicenses []utils.PackageLicense, disallowedLicense []string, disallowedPackage map[utils.PackageInfo]struct{}, disallowedPackageName map[string]struct{}) ([]utils.PackageLicense, []utils.PackageLicense) {
+	var violationLicense []utils.PackageLicense
+	var violationPackage []utils.PackageLicense
+
+	for _, packageInfo := range packageLicenses {
+		// if license contains disallowed, add to violation
+		for _, disallowed := range disallowedLicense {
```

Comment:
as we discussed before, license could be `ORed` or `ANDed` just like this [example](https://spdx.github.io/spdx-spec/v2.3/package-information/#7133-examples). I guess we'll handle the `AND` case first?